packages/alert/index.js

```diff
@@ -72,6 +49,23 @@ class WarpAlert extends WarpElement {
     `,
   ];
 
+  connectedCallback() {
+    super.connectedCallback();
+    if (!this.variant || !variants[this.variant]) {
+      throw new Error('Invalid "variant" attribute. Set its value to one of the following:\nnegative, positive, warning, info.');
+    }
+  }
+
+  get _wrapperClasses() {
+    return classNames([ccAlert.wrapper, ccAlert[this.variant]]);
+  }
+
+  get _iconClasses() {
+    const activeIconClassNames = ccAlert[`${this.variant}Icon`];
+
+    return classNames([ccAlert.icon, activeIconClassNames]);
+  }
+
```

Comment:
I'm trying to understand this change. Is there a pattern you were thinking to introduce in our web components' structure? If so, what would that be?
My concern here is that the more changes are baked into one feature branch (`component-classes-cleanup`), the more difficult it will be for us to spot if we've made a mistake in that branch when merging with `next` branch.

Reply:
Are you referring me moving this part below the static styles? I just moved them after static styles because I see that that is the structure we usually have in our web components, but I can revert that move. :)

Reply:
It does make sense to keep a certain order of stuff in our web components. Perhaps we could also keep static styles and static properties closer to each other, too, and make the constructor come after those and before other getters/setters/methods/properties?
I'm just wondering if it makes sense to have these sort of changes in separate PRs (as a general rule) to facilitate the review process of big-a** PRs like `component-classes-cleanup -> next`.